contributors.md

```diff
@@ -1,4 +1,4 @@
-# Contributors to OpenRCT2
+﻿# Contributors to OpenRCT2
```

Comment:
You appear to have accidentally added a byte-order mark to the file. Could you save it as plain UTF-8 (rather than UTF-8 with BOM) and recommit?